scenario/elasticexplore.go

```diff
@@ -187,7 +187,7 @@ func (settings ElasticExploreSettings) Execute(sessionState *session.State, acti
 		}
 	}
 	if space != nil && space.Links.Assignments.Href != "" {
-		sessionState.Rest.GetAsync(space.Links.Assignments.Href, actionState, sessionState.LogEntry, nil)
+		sessionState.Rest.GetAsync(space.Links.Assignments.Href, actionState, sessionState.LogEntry, nil) // Need to add ?limit=100 to space.Links.Assignments.Href
```

Comment:
sessionState.Rest.GetAsync(space.Links.Assignments.Href, actionState, sessionState.LogEntry, nil)
How exectly does the query in href look?
There's the "easy" way if we're sure how it looks with adding 
space.Links.Assignments.Href + "?limit=100" , if we know it doesn't already have any parameters
or space.Links.Assignments.Href + "&limit=100" 
to be sure to always work we could parse it to Url, add the parameter and then back to string.